app-multimedia/qmmp/autobuild/defines

```diff
@@ -1,6 +1,6 @@
 PKGNAME=qmmp
 PKGSEC=sound
-PKGDEP="curl libmad libogg libvorbis qt-5 taglib xdg-utils faad2 ffmpeg flac libcddb \
+PKGDEP="curl libmad libogg libvorbis qt-6 taglib xdg-utils faad2 ffmpeg flac libcddb \
```

Comment:
Please don't upgrade to Qt 6 yet. We do not have the proper theming engines (provided by KDE) and the program will have issue rendering file dialogues properly.

Please use the 1.6 branch for Qt 5 for now.

Reply:
Changed to use qmmp 1.6.5